src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Ball ball = new Ball();
+        ColorSupplier colorSupplier = new ColorSupplier();
+        String color = colorSupplier.getRandomColor();
+        ball.setColor(color);
+        Random random = new Random();
+        /*const MAX_BOUND = 101;
+        ball.setNumber(random.nextInt(MAX_BOUND));
+
+        Cannot resolve symbol 'constant'
+        */
```

Comment:
in Java, we dont have `const` keyword. 15 sec google actions from your side should be enough to get info about it:
https://chat.openai.com/share/6062b34b-3914-48e3-ae9f-fed6e7896f04
(but better use `private`, not `public`)